sparkly/utils.py

```diff
@@ -93,6 +181,45 @@ def kafka_get_topics_offsets(host, topic, port=9092):
     return offsets
 
 
+class lru_cache(object):
```

Comment:
It's not the same. The current approach can handle un-persisting the dataframes on eviction from cache. If we separate them that's not possible anymore.